UnitsNet.Serialization.JsonNet.Tests/UnitsNetJsonDeserializationTests.cs

```diff
@@ -98,6 +98,15 @@ public void NullValue_ExpectNullReturned()
             Assert.Null(deserializedNullMass);
         }
 
+        [Fact]
+        public void EmptyJsonValue_ExpectNullReturned()
```

Comment:
What happens when deserializing to `Mass`, not nullable? Can this be a separate test?

Reply:
It throws a Newtonsoft.Json.JsonSerializationException: 'No JSON content found and type 'UnitsNet.Mass' is not nullable. Path '', line 0, position 0.'

Reply:
Alright, let's add a test case asserting this exception type is thrown in that case.

Reply:
I added it